modules/apps/frontend-theme/kyoung-liferay-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key=" lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+    <title>${the_title} - ${company_name}</title>
+
+    <meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+    <#--  pt animation js for header -->

```

Comment:
Remove unnecessary comments unless they are descriptive and informative and provide clarity to the file/block.